pallets/subtensor/src/registration.rs

```diff
@@ -529,134 +529,36 @@ impl<T: Config> Pallet<T> {
         hash_as_vec
     }
 
-    pub fn hash_block_and_hotkey(block_hash_bytes: &[u8], hotkey: &T::AccountId) -> H256 {
+    #[allow(clippy::indexing_slicing)]
```

Comment:
Is this something you plan of tackling when you deal with Sparse matrix calculation i.e. https://github.com/orgs/opentensor/projects/27/views/9?pane=issue&itemId=59053066

Reply:
correct.